src/main/scala/org/deepdive/ddlog/DeepDiveLogCompiler.scala

```diff
@@ -116,8 +116,9 @@ class DeepDiveLogCompiler( program : DeepDiveLog.Program, config : DeepDiveLog.C
   // odd has happened.
   def resolveName( v : Variable ) : String = {
     v match { case Variable(v,relName,i) =>
-      if(attrNameForRelationAndPosition contains (relName,i)) {
-        attrNameForRelationAndPosition(relName,i)
+      val realRel = relName.replaceAll("^" + deepdivePrefixForVariablesWithIdsTable, "")
```

Comment:
I'm not comfortable with stitching regex like this.. Looks like [`stripPrefix`](http://www.scala-lang.org/api/2.10.x/index.html#scala.collection.immutable.StringOps@stripPrefix(prefix:String):String) will do a cleaner job?